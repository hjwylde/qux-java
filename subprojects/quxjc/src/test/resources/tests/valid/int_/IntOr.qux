package valid.int_

void main([str] args):
    print 0 or 1
    print 1 or 2 or 4
    print 0xff or 0xff
    print 1 or 1
    print 0 or 0
    print -1 or 0
    print -1 or 1
    print -1 or 2
    print 0xff or 0xffff
    print 0xffff or 0xff

