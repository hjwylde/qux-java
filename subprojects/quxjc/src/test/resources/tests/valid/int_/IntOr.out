1
7
255
1
0
-1
-1
-1
65535
65535
